### ISSUE TITLE
Boxes should be centered

### ISSUE THREAD

**Issue by nivthefox** [mephitrpg/rpg-cards#102]
When using the `boxes` key, the resulting boxes should be centered in the card so that the appearance is symmetrical.

**Comment by crobi** [comment]
I disagree - most of the other content is left-aligned. We could add alignment as an option though.
